packages/client/test/end-to-end/MemoryLeaks.test.ts

```diff
@@ -61,7 +61,7 @@ describe('MemoryLeaks', () => {
         expect(leaksDetector).toBeTruthy()
         if (!leaksDetector) { return }
         const detector = leaksDetector
-        await wait(1000)
+        await wait(5000)
```

Comment:
Increased wait needed?

Reply:
Could add a note to the PR description or add TODO so that it is easy to fix this later